content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -345,6 +345,21 @@ If your container has previously crashed, you can access the previous container'
 kubectl logs --previous ${POD_NAME} ${CONTAINER_NAME}
 ```
 
+Since v1.32, you can request a specific log stream of the affected container. This functionality requires the
+`PodLogsQuerySplitStreams` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
+However, only the server side changes have been merged so far; the [corresponding kubectl changes](https://github.com/kubernetes/kubernetes/pull/128159) are still under review.
+In the meantime, you can fetch the desired log stream using the following method:
+
+```shell
+kubectl get --raw "/api/v1/namespaces/default/pods/${POD_NAME}/log?container=${CONTAINER_NAME}&stream=Stderr"
+```
+
+Once kubectl is updated, you will be able to easily retrieve a specific log stream using:
+
+```shell
+kubectl logs --stream=stderr ${POD_NAME} ${CONTAINER_NAME}
+```
```

Comment:
Until there's more support, this advice isn't much use to a reader who wants to troubleshoot a Pod.
```suggestion
```